storybook/src/Omar/Atoms/Radio/Radio.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from "react";
```

Comment:
- The Radio input should not be checked/unchecked when clicked. This is not the normal behavior. Please Modify this adhering to the normal radio button behavior. 
- The radio should not be clickable when it is disabled. 
- Please add a label to the radio
- Please add tests to the radio with react-testing-library
-

Reply:
the disabled behavior works perfectly now.
The label has been added.
Tests have been added.

Reply:
Please adhere to 1 point. Make the radio behave as expected.

Reply:
The radio button now works like the normal behavior. it's not the most semantic way to do it but good for now.

Reply:
The radio button now works like the normal behavior.